sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -347,7 +347,9 @@ private[ui] class SqlStatsPagedTable(
         {formatDurationVerbose(duration)}
       </td>
       <td>
-        {info.statement}
+        <span class="description-input">
```

Comment:
does this ellipsis expand to show the full query when clicked on?
is it possible to add an "expand all" functionality?

Reply:
yes it shows full query when clicked. 
In jobs page , stage page also similar ellipsis is present. But we don't have expand all in those pages too.
@srowen could you suggest if expand all needs to added in all the places for consistency?

Reply:
I'd just match any similar other page. If there isn't an expand all on the jobs page, we don't need to add it here.

Reply:
I sometimes find it useful to be able to copy-paste the whole table with all query history out to a spreadsheet. Expand all would be useful for that. But I agree that it's not a core feature for this PR, and it's consistent with other tabs.
Maybe raise a followup JIRA with the idea to add an expand all like that to all tables of the UI?